test/test_distribution.py

```diff
@@ -1215,3 +1225,48 @@ def test_categorical_event_dim_conversion(batch_shape, event_shape):
         funsor.to_data(data, name_to_dim=name_to_dim))
     assert actual_log_prob.shape == expected_log_prob.shape
     assert_close(actual_log_prob, expected_log_prob)
+
+
+@xfail_if_not_implemented()
+@pytest.mark.parametrize("shape", [(), (4,), (3, 2)], ids=str)
+def test_power_transform(shape):
```

Comment:
We should also test the other `funsor.distribution.Distribution` methods, especially sampling.

I believe you could achieve that by just adding the following test case to the list of test cases in `test/test_distribution_generic.py`:
```py
DistTestCase(
    "backend_dist.TransformedDistribution(backend_dist.Exponential(rate=case.rate), [backend_dist.transforms.PowerTransform(0.5),])",  # noqa: E501
    (("rate", f"rand({batch_shape})"),),
    funsor.Real,
    xfail_reason="backend not supported" if get_backend() != "torch" else "",
)
```

Reply:
done